client/src/components/Button/index.js

```diff
@@ -1,16 +1,22 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import styles from './Style.module.css';
+import { ThemeContext } from '../../ThemeContext.js';
 
 const Button = (props) => {
   const { label, type = 'primary', disabled = false, onClick, width } = props;
 
+  const [darkMode, setDarkMode] = useContext(ThemeContext);
+
   const _styles = {
     width: width || 'auto'
   };
 
   return (
     <div
-      className={`${styles.button} ${disabled && styles.disabled}`}
+      className={`${styles.button} 
+      ${
+        disabled === true ? styles.disabled : darkMode === true ? styles.darkMode : styles.lightMode
+      } `}
```

Comment:
```suggestion
      className={`
          ${styles.button} 
          ${disabled && styles.disabled}  
          ${darkMode  ? styles.darkMode : styles.lightMode} 
      `}
```

Reply:
This prevents the button from being transparent as the styles of darkMode/lightMode are applied even when the button is disabled.